examples/worlds/set_model_state.sdf

```diff
@@ -248,5 +249,54 @@
         </model_state>
       </plugin>
     </model>
+
+    <model name="model_state_example_link_velocity">
+      <pose>2 0 1 0 0 0</pose>
+      <link name="box_link">
+        <pose>0.0 0.0 0.0 0 0 0</pose>
+        <inertial>
+        <inertia>
+            <ixx>5</ixx>
+            <ixy>0</ixy>
+            <ixz>0</ixz>
+            <iyy>5</iyy>
+            <iyz>0</iyz>
+            <izz>5</izz>
+          </inertia>
+          <mass>120.0</mass>
+        </inertial>
+        <visual name="box_visual">
+          <pose>0.0 0.0 0.0 0 0 0</pose>
+          <geometry>
+            <box>
+              <size>0.5 0.5 0.5</size>
+            </box>
+          </geometry>
+          <material>
+            <ambient>1.0 0.0 0.0 1</ambient>
+            <diffuse>1.0 0.0 0.0 1</diffuse>
+            <specular>1.0 0.0 0.0 1</specular>
+          </material>
+        </visual>
+        <collision name="box_collision">
+          <pose>0.0 0.0 0.0 0 0 0</pose>
+          <geometry>
+            <box>
+              <size>0.5 0.5 0.5</size>
+            </box>
+          </geometry>
+        </collision>
+      </link>
+      <plugin
+        filename="gz-sim-set-model-state-system"
+        name="gz::sim::systems::SetModelState">
+        <model_state>
+         <link_state name="box_link">
+           <linear_velocity>0.1 -0.1 0.1</linear_velocity>
+           <angular_velocity>1 2 1</angular_velocity>
```

Comment:
I recommend adding a `degrees` boolean attribute to `//link_state/angular_velocity` to match the behavior of the `//joint_state/position/@degrees` and `//joint_state/velocity/@degrees` attributes. Then it would be nice to have pair of example boxes with angular velocity expressed in both deg/s and rad/s

Reply:
got it, will make the changes